render-test/metadata.hpp

```diff
@@ -82,13 +88,33 @@ struct NetworkProbe {
     size_t transferred;
 };
 
+struct GfxProbe {
+    struct Memory {
+        int allocated = 0;
+        int peak = 0;
+    };
+
+    GfxProbe() = default;
+    GfxProbe(const mbgl::gfx::RenderingStats&);
+
+    int numDrawCalls = 0;
```

Comment:
nit: no need ` = 0`

Reply:
This will lead to uninitialized memory

Reply:
Of course GfxProbes could be initialized like `GfxProbe probe {}` but that's really dangerous error prone code imho.

Reply:
`GfxProbe() = default;` shall solve it for you

Reply:
I solved the issue for now by properly using aggregate initialization `GfxProbe probe {}` instead of more traditional `GfxProbe probe;` initialization